Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Networking;
+using Random = System.Random;
+
+public class NetworkObjectSpawner : NetworkBehaviour
+{
+    public NetworkObject prefabReference;
+    Random m_RandomGenerator = new Random();
```

Comment:
does this need to be a class variable? It seems you only use it in a method, could be declared locally

Reply:
I am not sure what you mean here, do you mean the random and that it should be declared in the if statement instead?